src/roadmap/release-checklist.md

```diff
@@ -159,12 +159,10 @@ When testing shows no further issues and the release chart is ready to ship, mak
 
 It's time to retag each individual Docker image with the 'official' `$WORKFLOW_RELEASE` value in the `deis` [quay.io](https://quay.io/organization/deis) org.
 
-To do so, simply go back to the directory where you checked out the `deis-workflow-group` repo
-and run the following two commands to retag the images:
+To do so, simply run the following `deisrel` command:
 
 ```console
-BRANCH="release-$WORKFLOW_RELEASE" make git-checkout-branch
-TAG=$WORKFLOW_RELEASE ORG="deis" make docker-tag docker-push
+deisrel docker retag $WORKFLOW_RELEASE --new-org="deis"
```

Comment:
Don't we also need to supply the `--ref release-$WORKFLOW_RELEASE` in this deisrel command so that the original tags are based on commit shas at HEAD of said branch?